hellabot.go

```diff
@@ -45,6 +47,8 @@ type Bot struct {
 	SSL           bool
 	SASL          bool
 	HijackSession bool
+	// Bot's prefix
+	Prefix *irc.Prefix
```

Comment:
The `Prefix` access is not thread safe. It needs to be protected by a mutex and probably not exported.

Reply:
Thread safety? Are we doing thousands of concurrent reads and writes a second? Not exported? Isn't it useful to give access to prefix?

I'm closing all my PR's and Issues and doing a hard fork of this library...

Nothing personal, I just need this functionality

Reply:
All data races are bugs waiting to happen. Sorry you're upset that I won't merge your buggy code.

Reply:
Yeah, I over reacted... But on my hard fork, KittyBot, I fixed it and also fixed all data races that were already present in this code base. It seems like no one ever ran this with -race